.release-notes/4180.md

```diff
@@ -0,0 +1,7 @@
+## Fix unwanted indentation in `StdStream.print`
+
+When printing via `StdStream.print` strings containing the null terminator, the standard library was printing the string until the null terminator and then padding the printed string with space characters until the string size was reached.
+
+That behavior was introduced in release 0.12.0, fixing a left-over from when Pony strings were null terminated.
+
+Now  `Stdstream.print` is effectively printing every character in the string plus an ending newline character.
```

Comment:
I'm not sure what this is supposed to mean. What is "effectively printing"? Does "effectively" here mean "more or less"? If yes, that would be a disturbing release note and I think it should be rewritten.

I would rewrite this, but I'm not sure what the intent is...